docs/releases.md

```diff
@@ -2,33 +2,47 @@
 - **DietPi-Software** | **VSCodium** :octicons-arrow-right-16: As alternative to MS VSCode without MS branding, telemetry and licensing, the VSCodium code editor, with support for various programming languages, including Java, JavaScript, Go, Node.js, Python and C++, has been added as software install option to DietPi.
 
-
 ### Changes / Improvements / Optimisations
 
-- **DietPi-LetsEncrypt**  :octicons-arrow-right-16: It is now possible to obtain certificates for multiple domains, given as comma-separated list, with the first domain used as primary webserver name.
-- **DietPi-LetsEncrypt**  :octicons-arrow-right-16:  It is now possible to obtain certificates without having a webserver installed. Certbot can start an own internal webserver for such case and the certificate can then used to manually enable HTTPS for other web applications, running on other ports than 80/443.
-- **DietPi-LetsEncrypt**  :octicons-arrow-right-16:  It is now possible to enable OCSP (Online Certificate Status Protocol) stapling for the obtained certificate. This allows clients to check if the certificate has been revoked, e.g. when the server has been compromised or the private key stolen.
-- **DietPi-LetsEncrypt** | **Lighttpd**  :octicons-arrow-right-16:  HTTPS is now enabled for IPv6 requests and the deprecated TLSv1.0 and TLSv1.1 are disabled from Debian Buster on. With the Lighttpd version shipped by Debian Stretch, those TLS versions cannot be disabled.
-- **DietPi-NordVPN**  :octicons-arrow-right-16:  The script reveived a major rework to allow being setup as ProtonVPN client and even with a custom OVPN config. For this reason it has been renamed to "DietPi-VPN", with the script moved to /boot/dietpi/dietpi-vpn and the console command alias changed to "dietpi-vpn" accordingly. Additionally a killswitch feature has been added which forcefully drops all WAN packets not sent through the VPN, when the connection got lost, until the VPN is disconnected manually/intentionally. Further is IPv6 now automatically disabled when the VPN connection is established. This is required to prevent IPv6 leaks as most publish VPN providers/servers do not support IPv6. When it has not been disabled before, IPv6 is re-enabled automatically once the VPN connection stops. Many thanks to @ravenclaw900 for doing this major rework: https://github.com/MichaIng/DietPi/pull/4180
-- **DietPi-Arr_to_RAM** :octicons-arrow-right-16: Support for Sonarr v3 and Radarr v3 has been added. On first link to RAM, a script "/mnt/dietpi_userdata/(sonarr|radarr|lidarr)/dietpi-arr_to_RAM.sh" is created, which allows updating the linked database backups via the programs "Custom Script" feature. Since v3, it is not possible anymore to pass arguments to custom script or call scripts inside the /boot directory at all, which broke the previous "/boot/dietpi/misc/dietpi-arr_to_RAM 2 (sonarr|radarr|lidarr)" calls in two ways.
```

Comment:
According to deepl.com, both commas are optional:
> Since v3 it is not possible anymore to pass arguments to user-defined scripts or to call scripts within the boot directory

But the first forth-and-back translation actually leads to:
> Since v3 it is no longer possible to pass arguments to user-defined scripts or to call scripts within the boot directory

which I prefer most.